plugins/foreman_openscap/0.7/index.md

```diff
@@ -343,6 +343,78 @@ If you find a bug, please file it in
 See the [troubleshooting section](/manuals/latest/index.html#7.2GettingHelp)
 in the Foreman manual for more info.
 
+### 7.1.1 Debugging foreman_scap_client
+
+Everything seems to be set up correctly but the reports are not showing up in UI? These are a few debugging steps:
+
+
+* Running with Katello
+
+When running with [Katello](/plugins/katello/index.html), make sure your host is registered as [Content Host](plugins/katello/nightly/user_guide/content_hosts/index.html). Consumer certificates are used for client authentication instead of Puppet certs.
+
+
+* Scans based on cronlines
+
+Scanning and report generation are based on cron. You can inspect all the cron lines in ```/etc/cron.d/foreman_scap_client_cron```. This file is managed by Puppet and any changes will be overwritten on next Puppet run.
```

Comment:
s/and any changes/and any manual changes/
s/next Puppet run/next Puppet agent run/